srs_data/tagging.py

```diff
@@ -11,13 +11,30 @@
 
 class ProcessTagsThatClientReturns(BaseTool):
 
+    """
+    Before running this class, the client's changes to tags must be
+    incorporated manually into the uniform tag sample, then saved locally.
+    These steps are enumerated in the README under the
+    `ProcessTagsThatClientReturns` sub-header.
+
+    With the updated uniform sample, this class performs three steps:
+    1. Overwrites the uniform sample that was placed in S3 by
```

Comment:
this step needs to be changed to:
1) pull the current sampling file in S3
2) update the sampling files from the tags in the input file
3) then overwrite the uniform sampling file in S3 with this retagged file from #2